tests/Functional/Fixtures/Symfony/AppCache.php

```diff
@@ -2,12 +2,17 @@
 
 namespace FOS\HttpCache\Tests\Functional\Fixtures\Symfony;
 
+use FOS\HttpCache\SymfonyCache\CacheInvalidationInterface;
 use FOS\HttpCache\SymfonyCache\EventDispatchingHttpCache;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpCache\HttpCache;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 
-class AppCache extends EventDispatchingHttpCache
+class AppCache extends HttpCache implements CacheInvalidationInterface
 {
+    // http://stackoverflow.com/questions/31877844/php-trait-exposing-a-method-and-interfaces
+    use EventDispatchingHttpCache {fetch as public eventTriggeringFetch;}
```

Comment:
yes, because of https://github.com/FriendsOfSymfony/FOSHttpCache/blob/master/src/SymfonyCache/CacheInvalidationInterface.php or more specifically because of https://github.com/FriendsOfSymfony/FOSHttpCache/blob/master/src/SymfonyCache/RefreshSubscriber.php